packages/vuetify/src/styles/utilities/_screenreaders.sass

```diff
@@ -0,0 +1,13 @@
+// Source: https://github.com/twbs/bootstrap/blob/master/scss/mixins/_screen-reader.scss
+
+.v-sr-only,
+.v-sr-only-focusable:not(:focus)
+  position: absolute !important
```

Comment:
sort alpha

Reply:
@johnleider Sorry, what do you mean by that?

Reply:
sort properties alphabetically

Reply:
Arh sorting alphabetically ofc!